api_client.py

```diff
@@ -154,6 +154,7 @@ def add_siem_logeer_handler(self, logdir):
             logging_handler = logging.handlers.SysLogHandler(
                 address, facility, socktype
             )
+            logging_handler.append_nul = False
```

Comment:
This has been made configurable in https://github.com/sophos/Sophos-Central-SIEM-Integration/pull/70